tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -25,12 +25,51 @@ namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
     internal sealed class OrdersControllerTests
     {
         [Test]
-        public void Constructor_NullRepository_Throws()
+        public void Constructor_Null_Service_Repository_ThrowsArgumentNullException()
```

Comment:
Would it be possible to use a TestCase here, as these three are very similar. If you dont think it's worth it then dont worry

Reply:
Torn on this one. I've made the change but not sure I like it as find it harder to read which test may have caused the exception. Will try it out as it does simplify the test by reducing the amount of code.